### PR TITLE
chore(app): set up app in protobuf repo

### PR DESCRIPTION
**Pull request by Yougigun**
Because

we are going to develop app backend.

This commit

sets up need initial file and change for app backend

### CODE REVIEW

File: app/app/v1alpha/app_private_service.proto [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  // Adds a tag to a given repository. Note that this operation is only
+  // intended to register the information of an *already created* tag. This
+  // method should be called as part of the content push operation, right after
+  // the [PUT Manifest](https://distribution.github.io/distribution/#put-manifest) has
+  // succeeded. The distribution registry won't hold data such as the push time
+  // or the tag digest, so `app-backend` will hold this information locally.
+  rpc CreateRepositoryTag(CreateRepositoryTagRequest) returns (CreateRepositoryTagResponse);
+
+  // Delete a repository tag.
+  rpc DeleteRepositoryTag(DeleteRepositoryTagRequest) returns (DeleteRepositoryTagResponse);
```
Comment (heiruwu) [inline app/app/v1alpha/app_private_service.proto:29]: I think we don't need these, its for docker image operation in artifact-backend

**Review by donch1989 — CHANGES_REQUESTED** [review]
Hi @Yougigun,
Please add the necessary `field_behavior` annotations according to [this guide](https://grpc-ecosystem.github.io/grpc-gateway/docs/mapping/customizing_openapi_output/#using-googleapifield_behavior). This will ensure that the OpenAPI generator produces the correct documentation.

File: app/app/v1alpha/app_private_service.proto (hunk 1 above)
Comment (Yougigun) [reply app/app/v1alpha/app_private_service.proto]: right. I have removed that part. please check again. 🙇‍♂️

**Review by heiruwu — APPROVED** [review]

File: app/app/v1alpha/app.proto [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+   This API is under development and, therefore, some of its entities and
+   endpoints are not implemented yet. This section aims to give context about the
+   current interface and how it fits in the App vision.
+
+   # App
+
+   The App domain is responsible of storing data that will later be used for
+   processing unstructured data. App will support the following types of
+   data:
+
```
Comment (donch1989) [inline app/app/v1alpha/app.proto:46]: These content is wrong copied?

Comment (Yougigun) [reply app/app/v1alpha/app.proto]: right. 🤣  just removed. help to review again. 🙇‍♂️

**Review by donch1989 — APPROVED** [review]

**Review by donch1989 — APPROVED** [review]

**Pull request merged**